website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+One team could run on GPU nodes while another uses storage nodes, with each having a different set of defaults.
+This approach could some day lead Karpenter to allow provisioners with different cloud providers on the same cluster or a control plane on one cloud and data plane on another.
+
+### Karpenter scheduler
+
+Besides creating nodes, Karpenter also works with the Kubernetes schedular to schedule pods.
+To avoid conflicts, Karpenter only schedules pods that the Kubernetes scheduler has marked unschedulable.
+When Karpenter finds unschedulable pods, it provisions the needed nodes and immediately binds the previously unschedulable pods to the new nodes.
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+
+### Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to work other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+Other cloud assets you can identify include security groups and capacity type.
+Choosing spot instances is one feature that is specific to AWS.
+See [AWS labels](/docs/cloud-providers/aws/) for details.
+
+### Kubernetes cluster autoscaler
+Like Karpenter, [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is
+designed to add nodes when requests come in to run pods that cannot be met by current capacity.
+Cluster autoscaler is part of the Kubenetes project, with implementations by most major Kubernetes cloud providers.
+By taking a fresh look at provisioning, Karpenter offers the following improvements over the older cluster autoscaler:
+
+* **Tuned to manage AWS EC2 instances**: 
+At the time cluster autoscaler was created, cloud providers, such as GCE, only had about a half dozen instance types.
+Cluster autoscaler was not built with the flexibility to handle hundreds of instance types or request EC2 spot instances.
+Karpenter has the ability to efficiently address the full range of instance types available through AWS.
+
+* **Adds scheduling**: Cluster autoscaler doesn’t bind pods to the nodes it creates.
+A node that Karpenter creates knows to run the bound pods immediately when the node comes up.
+It doesn't have to wait for the scheduler or for the node to become ready.
+It can start pulling the container images it needs immediately.
+This alone can save an extra 15 seconds of performance.
+
+### Constraints
```

Comment:
Didn't have time to make it through this section or below.

Reply:
Okay. I've responded to everything else to this point (or asked for further clarification).